ha/setup/ha_setup.py

```diff
@@ -223,6 +223,25 @@ def standby_node(self, node_name: str) -> None:
         if json.loads(standby_output).get("status") == STATUSES.FAILED.value:
             raise HaConfigException(f"Failed to put cluster in standby mode. Error: {standby_output}")
 
+    @staticmethod
+    def get_ios_instance(machine_id: str) -> int:
```

Comment:
Why is it showing up as new code? This function was already there. Probably you didn't pull the code before coding it?

Reply:
Ohh Ok.

Reply:
Fixed